health-check.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Script to check health of API. It sends a curl command to the health endpoint and checks
+# that the status code is 200.
+# Note: This script can only be used when the application is running inside a container.
+
+set -e
+
+echo "API Health Check..."
+
+CONTAINER_NAME="trusted-intermediary-router-1"
+CONTAINER_PASSED="PASS: Container is Running"
+CONTAINER_FAILED="FAIL: Container is Running"
+API_HEALTH_CHECK_PASSED="PASS: API Health Check"
+API_HEALTH_CHECK_FAILED="FAIL: API Health Check"
+
+is_container_running() {
+  if docker ps --format '{{.Names}}' | grep -Eq "^${CONTAINER_NAME}\$"; then
+    echo "$CONTAINER_PASSED"
+  else
+    echo "$CONTAINER_FAILED"
+    echo "$API_HEALTH_CHECK_FAILED"
+    exit 1
+  fi
```

Comment:
Remove "if container is running"

Reply:
Done. This will make the script more general and it can be use with other events